components/atom/checkbox/src/index.scss

```diff
@@ -24,7 +24,7 @@ $bd-atom-checkbox-is-intermediate: $bd-atom-checkbox--checked !default;
 $bdc-atom-checkbox-is-disabled: color-variation($c-gray, 1) !default;
 $bgc-checked-atom-checkbox-is-disabled: color-variation($c-gray, 3) !default;
 $bgc-checked-atom-checkbox-is-intermediate: $bgc-checked-atom-checkbox !default;
-$c-atom-checkbox-custom-is-disabled: $c-atom-checkbox !default;
+$c-atom-checkbox-custom-is-disabled: $c-gray !default;
```

Comment:
If you want to customize the color, you should do it in the theme of your vertical

$c-atom-checkbox: $c-gray

Reply:
Totally, maybe it was not complete in the description of my pr, but this would be the new default color of the icon when it is disabled for SUI components at the decision of @turolopezsanabria and UX. From there, as you comment, each vertical can change this color.

Thank you for your observation

Reply:
create a new token
````
$c-atom-checkbox-is-disabled: $c-gray !default;
```
and assign this token to the disabled one:
```
$c-atom-checkbox-custom-is-disabled: $c-gray !default;
```

then, modify, if necessary, in your vertical theme

Reply:
done @andresin87